library/src/precond/rocsparse_csrilu0.hpp

```diff
@@ -620,6 +690,7 @@ rocsparse_status rocsparse_csrilu0_dispatch(rocsparse_handle          handle,
             }
             else
             {
+#ifdef USE_ORIGINAL
```

Comment:
we are not using these type of macros.
you'll need to derive a new algorithm type.